pydicom/tests/test_dataset.py

```diff
@@ -247,6 +250,12 @@ def testGetDefault4(self):
                " for non-member by Tag")
         self.assertEqual(not_there, "not-there", msg)
 
+    def test_get_raises(self):
+        """Test Dataset.get() raises exception when invalid Tag"""
+        ds = self.dummy_dataset()
+        with pytest.raises(TypeError):
```

Comment:
I would also check if the message raised is the one that you are expecting.

You might want to reuse a utility as: https://github.com/glemaitre/specio/blob/master/specio/testing.py
which can be used as in https://github.com/glemaitre/specio/blob/master/specio/core/tests/test_functions.py#L32